OIIInspector/OIIIClient.py

```diff
@@ -1,9 +1,10 @@
-import json
+import os
 
 
 class OIIIClient:
 
-    def get_index_image_package_list(self, image_address, address_type):
-
-        print(image_address, address_type)
-        return json.loads("{}")
+    def get_index_image_package_list(self, image_address):
+        grp_curl_position = "./grpCurl/grpcurl"
+        command_to_call = grp_curl_position + " -plaintext " + image_address + ' list'
```

Comment:
Could you use string formatting? https://realpython.com/python-string-formatting/ Preferably option 2 or 3.